omod/src/main/webapp/resources/scripts/visit/filters.js

```diff
@@ -398,7 +398,7 @@ angular.module("filters", [ "uicommons.filters", "constants", "encounterTypeConf
 
             if (encounters) {
                 _.each(encounters, function(e) {
-                    if (EncounterTypeConfig.get(e.encounterType.uuid) && EncounterTypeConfig.get(e.encounterType.uuid).showOnVisitList
+                    if (EncounterTypeConfig.get(e) && EncounterTypeConfig.get(e).showOnVisitList
```

Comment:
changing the first parameter of the encounterTypeConfig.get so that it takes the full encounter, not the encounter type.

(Refactored get method coming in the config-pihemr PR)